finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ssl/server/Netty4SniHandler.scala

```diff
@@ -0,0 +1,75 @@
+package com.twitter.finagle.netty4.ssl.server
+
+import com.twitter.finagle.ssl.Engine
+import com.twitter.finagle.ssl.server.{SslServerConfiguration, SslServerEngineFactory, SslServerSessionVerifier}
+import com.twitter.finagle.transport.Transport
+import com.twitter.util.{Return, Throw}
+import io.netty.channel.ChannelHandlerContext
+import io.netty.handler.codec.DecoderException
+import io.netty.handler.ssl.{AbstractSniHandler, SniHandler, SslContext, SslHandler}
+import io.netty.util.{AsyncMapping, ReferenceCountUtil}
+import io.netty.util.concurrent.{Future, Promise}
```

Comment:
Please make this Future => NettyFuture. Since it's the Twitter codebase, Future within it almost always means Twitter Future.